Configuration/AlCa/python/autoCond.py

```diff
@@ -44,31 +44,31 @@
-    'phase1_2017_realistic'    : '103X_mc2017_realistic_v2',
+    'phase1_2017_realistic'    :  '105X_mc2017_realistic_v3',
     # GlobalTag for MC production (cosmics) with realistic alignment and calibrations for Phase1 2017 detector, Strip tracker in DECO mode
-    'phase1_2017_cosmics'      : '103X_mc2017cosmics_realistic_deco_v2',
+    'phase1_2017_cosmics'      :  '105X_mc2017cosmics_realistic_deco_v3',
     # GlobalTag for MC production (cosmics) with realistic alignment and calibrations for Phase1 2017 detector, Strip tracker in PEAK mode
-    'phase1_2017_cosmics_peak' : '103X_mc2017cosmics_realistic_peak_v2',
+    'phase1_2017_cosmics_peak' :  '105X_mc2017cosmics_realistic_peak_v4',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for full Phase1 2018 (and 0,0,0-centred beamspot)
-    'phase1_2018_design'       : '103X_upgrade2018_design_v4',
+    'phase1_2018_design'       :  '105X_upgrade2018_design_v2',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector
-    'phase1_2018_realistic'    : '103X_upgrade2018_realistic_v8',
+    'phase1_2018_realistic'    :  '105X_upgrade2018_realistic_v2',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector for Heavy Ion
-    'phase1_2018_realistic_hi' : '103X_upgrade2018_realistic_HI_v11',
+    'phase1_2018_realistic_hi' :  '103X_upgrade2018_realistic_HI_v11',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector: HEM-15-16 fail
-    'phase1_2018_realistic_HEfail'    : '103X_upgrade2018_realistic_HEfail_v4',
+    'phase1_2018_realistic_HEfail' :  '105X_upgrade2018_realistic_HEfail_v3',
     # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in DECO mode
-    'phase1_2018_cosmics'      :   '103X_upgrade2018cosmics_realistic_deco_v7',
+    'phase1_2018_cosmics'      :   '105X_upgrade2018cosmics_realistic_deco_v3',
     # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in PEAK mode
-    'phase1_2018_cosmics_peak' :   '103X_upgrade2018cosmics_realistic_peak_v1',
+    'phase1_2018_cosmics_peak' :   '105X_upgrade2018cosmics_realistic_peak_v3',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2019
-    'phase1_2019_design'       : '103X_postLS2_design_v4', # GT containing design conditions for postLS2
+    'phase1_2019_design'       : '105X_postLS2_design_v1', # GT containing design conditions for postLS2
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2019
-    'phase1_2019_realistic'    : '103X_postLS2_realistic_v4', # GT containing realistic conditions for postLS2
+    'phase1_2019_realistic'    : '105X_postLS2_realistic_v2', # GT containing realistic conditions for postLS2
```

Comment:
Hi @tocheng, 
I am a trifle confused by the last change about reverting the MC misalignment scenario in the 2019 queue:
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/105X_postLS2_realistic_v1/105X_postLS2_realistic_v2,
`TrackerAlignment_Upgrade2017_realistic_v2`  contains an unintentionally large weak mode and should therefore be removed from all autoCond GTs.
The "spurious changes" I mentioned in https://github.com/cms-sw/cmssw/pull/25885#issuecomment-462333444 were spurious only in the context of PR #25885 which was meant to be technical and not change physics output. As you are changing conditions for real here, I would prefer to keep the fixed alignment. Thanks

Reply:
OK. I agree.